app/main.go

```diff
@@ -92,6 +92,9 @@ var (
 	boltdbPath = kingpin.Flag("boltdb-path", "path to BoltDB store, e.g. /data/bolt.db").Envar("BOLTDB_PATH").String()
 
 	ver = kingpin.Flag("version", "print the current version revision").Bool()
+
+	monitoringBackends = kingpin.Flag(
```

Comment:
Nit: Workflow here is a bit confusing. Actual help doesn't specify which options are available for backends, e.g.:
```
  --metric-exporters=METRIC-EXPORTERS ...
                                Monitoring backend(s) for internal metrics.
```
, specifying something incorrectly does, however:

```
docker run -it ts-bridge:e1df8f6 --metric-exporters=stackdrivr
ts-bridge: error: enum value must be one of prometheus,stackdriver, got 'stackdrivr', try --help
```

Can you see if it's possible to fix?

Reply:
This appears to be a kingpin limitation, I've set stackdriver as the default for now.